### PR TITLE
MCOL-4157 mcs-loadbrm.py doesn't start SM with systemd with a certain…

### PR DESCRIPTION
**Pull request by drrtuy**
… argument

This feature is used to start multi-node MCS in containers

### CODE REVIEW

File: oam/install_scripts/mcs-loadbrm.py.in [hunk 1]
```diff
@@ -28,6 +28,10 @@ def get_port():
 
 
 if __name__ == '__main__':
+    # To avoid systemd in container environment
+    use_systemd = True
+    if len(sys.argv) > 1:
+        use_systemd = not sys.argv[1] == 'no'
```
Comment (jmrojas2332) [inline oam/install_scripts/mcs-loadbrm.py.in:34]: Should the argument being passed in be more descriptive?
It is not clear what ` mcs-loadbrm.py no` would do w/o digging into mcs-loadbrm.py.

**Review by jmrojas2332 — CHANGES_REQUESTED** [review]
Just a comment about argument being passed in. Logic looks good.

File: oam/install_scripts/mcs-loadbrm.py.in (hunk 1 above)
Comment (drrtuy) [reply oam/install_scripts/mcs-loadbrm.py.in:34]: There should be getopt org ArgParse used here but I don't want to bother with them having a single argument possible. If we have more then we'll need to use one of those modules.

Comment (drrtuy) [reply oam/install_scripts/mcs-loadbrm.py.in:34]: There is a comment here already and another one must go into the corresponding CMAPI PR.

Comment (jmrojas2332) [reply oam/install_scripts/mcs-loadbrm.py.in:34]: Okay, if there is one in the CMAPI PR then that's fine.

**Review by jmrojas2332 — APPROVED** [review]
Looks good to me.

**Pull request merged**